app/helpers/script_helper.rb

```diff
@@ -31,14 +31,18 @@ def render_javascript_pack_once_tags(*names)
 
   private
 
+  SAME_ORIGIN_ASSETS = %w[
+    identity-style-guide/dist/assets/img/sprite.svg
+  ]
```

Comment:
```suggestion
  ].to_set.freeze
```

Reply:
One of these days I'll remember to freeze my constants 😅